<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Lblmnemonic_id">
       <property name="text">
        <string>mnemonic_id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lblmnemonic">
       <property name="text">
        <string>mnemonic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lblunit">
       <property name="text">
        <string>unit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lbldescription">
       <property name="text">
        <string>description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lbltypedevice_id">
       <property name="text">
        <string>typedevice_id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lblparent_mnemonic_id">
       <property name="text">
        <string>parent_mnemonic_id</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lbmnemonic_id"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lbmnemonic"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lbunit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lbdescription"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lbtypedevice_id"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lbparent_mnemonic_id"/>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog2</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog2</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
